src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -2066,11 +2066,10 @@ HRESULT CEndlessUsbToolDlg::OnLiveUsbClicked(IHTMLElement* pElement)
 {
     FUNCTION_ENTER;
 
-	SetSelectedInstallMethod(InstallMethod_t::LiveUsb);
+    SetSelectedInstallMethod(InstallMethod_t::LiveUsb);
```

Comment:
This is deliberate-ish. When I wrote the [.editorconfig](https://github.com/endlessm/rufus/blob/master/.editorconfig#L7-L10) file, I checked and this file had pretty much equal spaces and tabs. I try to be consistent with what's already there but err on the side of spaces. In these cases, these 3-4 line functions already had inconsistent whitespace (the `FUNCTION_ENTER`s are space-indented, the other stuff tab-indented) so I decided to fix them while I was there, but this is lazy of me… I'll rework the patch.